spring-messaging/src/test/java/org/springframework/messaging/simp/stomp/SplittingStompEncoderTests.java

```diff
@@ -0,0 +1,382 @@
+/*
+ * Copyright 2024-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.messaging.simp.stomp;
+
+import java.io.ByteArrayOutputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+/**
+ * Unit tests for {@link SplittingStompEncoder}.
+ *
+ * @author Injae Kim
+ * @since 6.2
+ */
+public class SplittingStompEncoderTests {
```

Comment:
> Reply to https://github.com/spring-projects/spring-framework/pull/31970#pullrequestreview-1811109435
> I think it would be good to have dedicated unit tests for SplittingStompEncoder in spring-messaging in addition to the those in spring-websocket.

Hi @rstoyanchev ! thanks for your quick review.

I already added this `SplittingStompEncoderTests` unit test on `spring-messaging` for `SplittingStompEncoder`.
You mean we need more unit test for `SplittingStompEncoderTests`?